### PR TITLE
added responsive navigation component

### PR DESCRIPTION
**Pull request by pankaj-prog**
navigation look like this :
in large screens : 
 
res nav 2

in small screens:
res nav

### CODE REVIEW

File: src/components/navigation/navigation.css [hunk 1]
```diff
@@ -60,3 +60,24 @@
...
+@media screen and (max-width: 1200px) {
+  .hide-mobile {
+    display: none;
+  }
+
+  .hide-desktop {
+    display: block;
+  }
+  .header-search-wrapper {
+    flex-grow: 1;
```
Comment (manojsarna) [inline src/components/navigation/navigation.css:81]: do cover more screen sizes for media queries

File: src/components/navigation/navigation.html [hunk 2]
```diff
@@ -146,6 +146,70 @@ <h2 class="component-type-name">1. Simple Navigation</h2>
...
+            <nav id="header-nav" class="header-nav hide-mobile">
+              <ul>
+                <li><a href="#">Men</a></li>
+                <li><a href="#">Women</a></li>
+                <li><a href="#">Kids</a></li>
+              </ul>
+            </nav>
+            <div class="header-search-wrapper">
+              <label for="search"><i class="fas fa-search"></i></label>
+              <input type="text" id="search" />
```
Comment (manojsarna) [inline src/components/navigation/navigation.html:167]: rest is looking good!!

**Review by manojsarna — APPROVED** [review]
Do Check the Comments !!

File: src/components/navigation/navigation.css [hunk 3]
```diff
@@ -60,3 +60,24 @@
...
+
+.hamburger-icon-btn i {
+  font-size: 1rem;
+}
+
+.hide-desktop {
+  display: none;
+}
+
+@media screen and (max-width: 1200px) {
```
Comment (theVivek-kumar) [inline src/components/navigation/navigation.css:72]: if u have time add media for more screen size

File: src/components/navigation/navigation.html [hunk 4]
```diff
@@ -146,6 +146,70 @@ <h2 class="component-type-name">1. Simple Navigation</h2>
...
+          This navigation consist is similar to the desktop navigation with
+          hamburger icon for menu on smaller screens.
+        </p>
+        <section class="component-example-wrapper">
+          <header class="header">
+            <a href="#" class="header-brand h3">E-Com</a>
+            <nav id="header-nav" class="header-nav hide-mobile">
+              <ul>
+                <li><a href="#">Men</a></li>
+                <li><a href="#">Women</a></li>
```
Comment (theVivek-kumar) [inline src/components/navigation/navigation.html:161]: code looks very clean to me

**Review by theVivek-kumar — APPROVED** [review]
just an suggestion if u have time add more screen size rest looks good

File: src/components/navigation/navigation.css (hunk 3 above)
Comment (pankaj-prog) [reply src/components/navigation/navigation.css:72]: okay sure

File: src/components/navigation/navigation.css (hunk 1 above)
Comment (pankaj-prog) [reply src/components/navigation/navigation.css:81]: sure

**Pull request merged**